### ISSUE TITLE
The failure of the save

### ISSUE THREAD

**Issue by DDA-shnik** [CleverRaven/Cataclysm-DDA#71193]
### Describe the bug

I took a task from the Valhallists to rescue 3 members of their group. When I got to my destination, I talked to each one in turn, after which they followed me. On the way to the Valhallist base, one of the group members was killed and immediately after that the game crashed. An error appears when trying to load the last save. The error appears after successful file upload, completion, and verification. It does not respond to pressing [i] and [space]. This is the mistake:

 DEBUG    : parent location does not point to valid item

 FUNCTION : deserialize
 FILE     : D:\a\Cataclysm-DDA\Cataclysm-DDA\src\item_location.cpp
 LINE     : 812
 VERSION  : d6ec466

### Attach save file

Unfortunately, due to a save failure, I don't have access to the debug menu.
[Primhar.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/14024226/Primhar.zip)
[Uploading Primhar1-2.zip…]()
[Uploading Primhar3-4.zip…]()
[Uploading Primhar5-6.zip…]()

### Steps to reproduce

Take a task from the Valhallists in the center, get to the marked place, talk to all the NPCs in turn, let one of them die.

### Expected behavior

I expected that I would simply fail to complete the task, but apparently something went wrong.

### Screenshots

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/157487163/122618f0-ca95-40e8-80ff-c7ef6204e6b7)

### Versions and configuration

OC: Windows 10
Version: 0.G, d6ec466
Language: Russian/Русский
Tileset: Chibi_Ultica
Download mods:
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/157487163/0afe198b-b204-434c-a3d4-b4eb0419fc3f)

### Additional context

[debug.log](https://github.com/CleverRaven/Cataclysm-DDA/files/14024161/debug.log)

**Comment by Karol1223** [comment]
Given the Valhallists have been removed in the experimental cycle, this bug should have been removed with them.

**Comment by RenechCDDA** [comment]
The Valhallists seem to be tangential as far as I can tell.

@DDA-shnik Your save did not finish uploading before you submitted, can you try again?

Also, can you provide crash.log(if it exists) and describe the crash? Was there any error during the crash, did it happen during an autosave, did it happen *on the same turn* as the NPC died? Any information you can provide is helpful, but my working theory is that you CTD crashed during an autosave. You might be able to load your save in newer experimental versions (it tries to force it through even with bad data like this), but I can't promise it will work flawlessly.

**Comment by DDA-shnik** [comment]
Yes, the flight took place in the same turn after the NPC was killed
